packages/e2e-test-utils/src/widgets.js

```diff
@@ -1,32 +1,36 @@
 /**
  * Internal dependencies
  */
-import { activatePlugin } from './activate-plugin';
-import { deactivatePlugin } from './deactivate-plugin';
-import { visitAdminPage } from './visit-admin-page';
+import { rest } from './rest-api';
 
 /**
  * Delete all the widgets in the widgets screen.
  */
 export async function deleteAllWidgets() {
-	// TODO: Deleting widgets in the new widgets screen is cumbersome and slow.
-	// To workaround this for now, we visit the old widgets screen to delete them.
-	await activatePlugin( 'gutenberg-test-classic-widgets' );
+	const [ widgets, sidebars ] = await Promise.all( [
+		rest( { path: '/wp/v2/widgets' } ),
+		rest( { path: '/wp/v2/sidebars' } ),
+	] );
 
-	await visitAdminPage( 'widgets.php' );
+	await rest( {
+		method: 'POST',
+		path: '/batch/v1',
+		data: {
+			requests: widgets.map( ( widget ) => ( {
+				method: 'DELETE',
+				path: `/wp/v2/widgets/${ widget.id }?force=true`,
+			} ) ),
+			validation: 'require-all-validate',
+		},
+	} );
 
-	let widget = await page.$( '.widgets-sortables .widget' );
-
-	// We have to do this one-by-one since there might be race condition when deleting multiple widgets at once.
-	while ( widget ) {
-		const deleteButton = await widget.$( 'button.widget-control-remove' );
-		const id = await widget.evaluate( ( node ) => node.id );
-		await deleteButton.evaluate( ( node ) => node.click() );
-		// Wait for the widget to be removed from DOM.
-		await page.waitForSelector( `#${ id }`, { hidden: true } );
-
-		widget = await page.$( '.widgets-sortables .widget' );
-	}
-
-	await deactivatePlugin( 'gutenberg-test-classic-widgets' );
+	await Promise.all(
```

Comment:
Why can't we use `/batch/v1` for this as well?

Reply:
It seems like `GET` requests are not supported for security reasons:

https://make.wordpress.org/core/2020/11/20/rest-api-batch-framework-in-wordpress-5-6/#comment-40916